miio/airconditioningcompanion.py

```diff
@@ -109,3 +160,40 @@ def send_command(self, command: str):
 
         :param str command: Command to execute"""
         return self.send("send_cmd", [str(command)])
+
+    def send_configuration(self, model: str, power: Power,
+                           operation_mode: OperationMode,
+                           target_temperature: float, fan_speed: FanSpeed,
+                           swing_mode: SwingMode):
+
+        # Static turn off command available?
+        if (power is False) and (model in DEVICE_COMMAND_TEMPLATES) and \
+                (POWER_OFF in DEVICE_COMMAND_TEMPLATES[model]):
+            return self.send_command(
+                model + DEVICE_COMMAND_TEMPLATES[model][POWER_OFF])
+
+        if model in DEVICE_COMMAND_TEMPLATES:
+            configuration = model + DEVICE_COMMAND_TEMPLATES[model]['base']
+        else:
+            configuration = model + DEVICE_COMMAND_TEMPLATES['fallback']['base']
+
+        configuration = configuration.replace('po', power.value)
```

Comment:
Just a suggestion, would it be better to replace placeholders with something like `[PO]` to make it more clear that they are placeholders? Just a thought though, otherwise this seems fine to me so please feel to merge when you want.

Reply:
Good point!